packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -582,6 +598,7 @@ class _SelectableTextState extends State<SelectableText> with AutomaticKeepAlive
         autofocus: widget.autofocus,
         forceLine: false,
         toolbarOptions: widget.toolbarOptions,
+        minLines: widget.minLines,
```

Comment:
Not to be addressed here, but I just noticed that `DefaultTextStyle` does not have a `minLines` parameter :)

Reply:
Yes, is that a different issue/feature?

Reply:
Yeah, you should probably open a new issue for that.